src/main/java/dev/nolij/zson/ZsonValue.java

```diff
@@ -10,9 +11,14 @@
 public final class ZsonValue {
 
 	/**
-	 * The comment for this value. If null, there is no comment.
+	 * The value for {@link #comment} when there is no comment, represented as the null character.
 	 */
-	@Nullable
+	public static final String NO_COMMENT = "\0";
```

Comment:
Does Java allow this? I feel like this might just end up as an empty string...

Reply:
technically `"\0".length() == 1` and `"\0".isEmpty() == false`